src/components/DependencyManagementPane/DependencyManagementPane.js

```diff
@@ -131,6 +132,17 @@ class DependencyManagementPane extends PureComponent<Props, State> {
                           : undefined
                       }
                     />
+                  ) : dependency.status.match(/^queued-/) ? (
+                    <span
+                      style={{
+                        color:
+                          selectedDependencyIndex === index
+                            ? COLORS.white
+                            : undefined,
+                      }}
+                    >
+                      <IconBase size={20} icon={moreHorizontal} />
+                    </span>
```

Comment:
Agreed, I was also afraid of the confusion around the dots. I'm coming at all of this from the perspective of a developer, and I always want to know exactly what is happening, but from the end-user standpoint it definitely makes more sense to simplify it since they have no control over the process.